### DISCUSSION TITLE
Full Dreambooth SDXL on Runpod?

### DISCUSSION THREAD

**Discussion by rycossette** [TheLastBen/fast-stable-diffusion#2467] [General]
Thanks for the Lora Dreambooth on Runpod, it works pretty well. I am hoping you have plans for a full dreambooth for SDXL on runpod as well. I am hoping to have a base model trained on my own artwork, and then train LORAs on specific characters. I've not had luck trying to combine multiple loras in one prompt, so having a base model is ideal.

**Comment by TheLastBen** [comment]
The results would be the same with full dreambooth or LoRA, the key is to not overtrain your LoRAs to allow mixing and flexibility, keep the text encoder step to a minimum, a total steps of 100 would suffice, the learning rates should not go above 1e-6, this way, you'll be able to use multiple loras without having bad results

**Reply by rycossette** [reply]
awesome thanks, I had no idea

**Reply by KodyAG** [reply]
Is it possible to download and use other sdxl models on runpod, and how to do that?

**Comment by Taikakim** [comment]
Hey, one thing, just to check, so the LoRA template does resume when retraining a model? My results have been a bit iconsistent (I have a challenging subject though), so I'd just like to make sure it does, so I'm not just training more and more models of 20 epochs when I'm thinking I'm making a gradually progressing training :)
